src/const.ts

```diff
@@ -0,0 +1,17 @@
+export const RLN_ABI = [
+  "function MEMBERSHIP_DEPOSIT() public view returns(uint256)",
+  "function register(uint256 pubkey) external payable",
+  "function withdraw(uint256 secret, uint256 _pubkeyIndex, address payable receiver) external",
+  "event MemberRegistered(uint256 pubkey, uint256 index)",
+  "event MemberWithdrawn(uint256 pubkey, uint256 index)",
+];
+
+export const DEV_CONTRACT = {
```

Comment:
```suggestion
export const GOERLI_CONTRACT = {
```
So it's clear it's the Goerli contract.